inference-engine/src/preprocessing/cpu_x86_avx2/ie_preprocess_gapi_kernels_avx2.cpp

```diff
@@ -131,6 +131,315 @@ void calcRowArea_32F(float dst[], const float *src[], const Size& inSz,
     calcRowArea_impl(dst, src, inSz, outSz, yalpha, ymap, xmaxdf, xindex, xalpha, vbuf);
 }
 
+static inline void main_computation_horizontalPass_lpi4(const v_uint8& val_0,
```

Comment:
8U?

Reply:
This name is enough long. If I add 8U, name of function will become too long. Moreover 32F will not have analogue.